convert2rhel/toolopts.py

```diff
@@ -49,6 +60,58 @@ def __init__(self):
         # set True when credentials (username & password) are given through CLI
         self.credentials_thru_cli = False
 
+    @staticmethod
+    def options_from_config_files(cfg_path=None):
```

Comment:
In other languages, like Java, it is common to make staticmethods but in Python, it's better to create a toplevel function rather than a `@staticmethod` most of the time.

The difference stems from the fact that Python builds namespaces out of each file on the filesystem whereas other programming languages depend on the class to be part of the namespace.  So I'd move this to just be a regular toplevel function in toolopts.py.